### PR TITLE
_projects: Update status of GSoC 2018 projects

### PR DESCRIPTION
**Pull request by 123vivekr**
This updates the status of GSoC 2018 projects in [projects.coala.io](https://projects.coala.io/#/projects)

Closes https://github.com/coala/projects/issues/636

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: _projects/gitmate_jira.md [hunk 1]
```diff
@@ -43,7 +42,6 @@ requirements:
   - "The participant should have at least one patch to
      [GitMate](https://gitlab.com/gitmate/open-source/gitmate-2) accepted."
   - "The participant should demonstrate that he can operate Jira from the Python shell in his proposal."
-status: in_progress
```
Comment (jayvdb) [inline _projects/gitmate_jira.md]: @nkprince007 , what is the status of this project?

File: _projects/editor_support.md [hunk 2]
```diff
@@ -1,11 +1,9 @@
 ---
 collaborating_projects:
   - coala
-status: in_progress
 desc: "Develop an efficient strategy that allows coala to maintain a number of plugins for different editors."
 difficulty: medium
 initiatives:
-  - GSoC
```
Comment (jayvdb) [inline _projects/editor_support.md]: please re-add GSoC here.

What I meant by 'non-GSoC' was that it had been marked as 'in progress', but no gsoc project had ever commenced, and I wasnt aware of who might have been doing it.

I've double checked now, and this project is certainly available to be worked on.

File: _projects/git_task_list.md [hunk 3]
```diff
@@ -17,9 +17,7 @@ requirements:
   - "The applicant should have at least one patch merged to any of the coala repositories."
   - "Familiarity with Git, GitHub API, Phabricator API, and GitLab API"
   - "Familiarity with EmberJS and GitHub Pages"
-status:
-  - in_progress
-  - disabled
+status: disabled
```
Comment (jayvdb) [inline _projects/git_task_list.md]: please undo this change.

Someone else has already marked it as disabled, and it is also 'in progress' - this is a strange one ;-)

File: _projects/gitmate_jira.md (hunk 1 above)
Comment (nkprince007) [reply _projects/gitmate_jira.md]: This project never surfaced completion. During the development of this project, we got the award from Mozilla Open Source Support and so, we shifted our focus towards finishing MOSS stuff.

Comment (nkprince007) [reply _projects/gitmate_jira.md]: Well, more like stopped amidst. No one seems to care about adding support to Jira because it involves having a lot of stuff to develop. First off, it requires our own instance of Jira to begin with.

**Comment by 123vivekr** [comment]
@jayvdb Requested changes have been applied. Commit message has been fixed and now abides to the Commit Message Guidelines.

File: _projects/gitmate_jira.md (hunk 1 above)
Comment (123vivekr) [reply _projects/gitmate_jira.md]: Ok, I'll change the project status to "available". Is that okay?

Comment (nkprince007) [reply _projects/gitmate_jira.md]: Yes.

File: _projects/gitmate_jira.md [hunk 4]
```diff
@@ -7,7 +7,6 @@ desc: "GitMate.io helps by automating tasks on GitHub and GitLab. It's time to a
 developers_involved: []
 difficulty: medium
 initiatives:
-  - GSoC
```
Comment (jayvdb) [inline _projects/gitmate_jira.md]: as this project is available, we need to keep 'GSoC' here

**Comment by 123vivekr** [comment]
@jayvdb GSoC added to the project.

**Comment by 123vivekr** [comment]
@jayvdb All the requested changes are done. But TravisCI tests are failing.
How do I fix the TravisCI test failing issue? 
[docutils.sourceforge.net](http://docutils.sourceforge.net/) is returning error code 503: No backend servers available

**Comment by 123vivekr** [comment]
hi

**Comment by 123vivekr** [comment]
Travis CI build is passing. Please review :)

**Comment by jayvdb** [comment]
Please do not close PRs.

**Comment by jayvdb** [comment]
ack https://github.com/coala/projects/pull/651/commits/0cf6b490a56f8a5eb5be862151f26a90f0106364

**Comment by jayvdb** [comment]
@gitmate-bot ff
